ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayAnnotation/DisplayAnnotation.h

```diff
@@ -0,0 +1,56 @@
+// [WriteFile Name=DisplayAnnotation, Category=Layers]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef DISPLAYANNOTATION_H
+#define DISPLAYANNOTATION_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class AnnotationLayer;
+class Map;
+class MapQuickView;
+}
+}
+
+#include <QObject>
+
+class DisplayAnnotation : public QObject
+{
+    Q_OBJECT
```

Comment:
Our coding standards are set to 2 spaces for tabs, as opposed to 4. This can be changed in the QtCreator IDE settings!